manifests/init.pp

```diff
@@ -1,3 +1,135 @@
+# @param max_filedescriptors 
+#   Defaults to undef. http://www.squid-cache.org/Doc/config/max_filedescriptors/
+# @param workers 
+#   Defaults to undef. http://www.squid-cache.org/Doc/config/workers/
+# @param snmp_incoming_address 
+#   Defaults to undef. Can be set to an IP address to only listen for snmp requests on an individual interface. http://www.squid-cache.org/Doc/config/snmp_incoming_address/
+# @param buffered_logs 
+#   Defaults to undef. http://www.squid-cache.org/Doc/config/buffered_logs/
+# @param acls 
+#   Defaults to undef. If you pass in a hash of acl entries, they will be defined automatically. http://www.squid-cache.org/Doc/config/acl/
+# @param visible_hostname 
+#   Defaults to undef. http://www.squid-cache.org/Doc/config/visible_hostname/
+# @param via 
+#   Defaults to undef. http://www.squid-cache.org/Doc/config/via/
+# @param httpd_suppress_version_string 
+#   Defaults to undef. http://www.squid-cache.org/Doc/config/httpd_suppress_version_string/
+# @param forwarded_for 
+#   Defaults to undef. supported values are "on", "off", "transparent", "delete", "truncate". http://www.squid-cache.org/Doc/config/forwarded_for/
+# @param http_access 
+#   Defaults to undef. If you pass in a hash of http_access entries, they will be defined automatically. http://www.squid-cache.org/Doc/config/http_access/
+# @param http_ports 
+#   Defaults to undef. If you pass in a hash of http_port entries, they will be defined automatically. http://www.squid-cache.org/Doc/config/http_port/
+# @param https_ports 
+#   Defaults to undef. If you pass in a hash of https_port entries, they will be defined automatically. http://www.squid-cache.org/Doc/config/https_port/
+# @param icp_access 
+#   Defaults to undef. If you pass in a hash of icp_access entries, they will be defined automatically. http://www.squid-cache.org/Doc/config/icp_access/
+# @param refresh_patterns 
+#   Defaults to undef.  If you pass a hash of refresh_pattern entires, they will be defined automatically. http://www.squid-cache.org/Doc/config/refresh_pattern/
+# @param snmp_ports 
+#   Defaults to undef. If you pass in a hash of snmp_port entries, they will be defined automatically. http://www.squid-cache.org/Doc/config/snmp_port/
+# @param send_hit 
+#   Defaults to undef. If you pass in a hash of send_hit entries, they will be defined automatically. http://www.squid-cache.org/Doc/config/send_hit/
+# @param cache_dirs 
+#   Defaults to undef. If you pass in a hash of cache_dir entries, they will be defined automatically. http://www.squid-cache.org/Doc/config/cache_dir/
+# @param ssl_bump 
+#   Defaults to undef. If you pass in a hash of ssl_bump entries, they will be defined automatically. http://www.squid-cache.org/Doc/config/ssl_bump/
+# @param sslproxy_cert_error 
+#   Defaults to undef. If you pass in a hash of sslproxy_cert_error entries, they will be defined automatically. http://www.squid-cache.org/Doc/config/sslproxy_cert_error/
```

Comment:
Here the default is somewhat useful:
```
##### `sslproxy_cert_error`

Data type: `Optional[Hash]`

Defaults to undef. If you pass in a hash of sslproxy_cert_error entries, they will be defined automatically. http://www.squid-cache.org/Doc/config/sslproxy_cert_error/
```

However, we can also move all the static values from `params.pp` to `init.pp` and remove it. That means it's clearer in the docs and less to maintain. Static copies of defaults can go out of sync.